From 7f24b35d7367f2617306e7af92e6aecf52974096 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Mon, 6 May 2019 14:56:27 +0700
Subject: [PATCH 091/248] arm64: dts: renesas: Extract sub board for common
 Hihope RZ/G2M and RZ/G2N support

The HiHope RZ/G2M and HiHope RZ/G2N boards are very similar.
To avoid duplication, prepare for the advent of the latter by
extracting the common parts of sub boards into its own .dtsi file.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/hihope-rzg2-ex.dtsi          | 10 ++++++++++
 arch/arm64/boot/dts/renesas/r8a774a1-hihope-rzg2m-ex.dts |  1 +
 2 files changed, 11 insertions(+)
 create mode 100644 arch/arm64/boot/dts/renesas/hihope-rzg2-ex.dtsi

diff --git a/arch/arm64/boot/dts/renesas/hihope-rzg2-ex.dtsi b/arch/arm64/boot/dts/renesas/hihope-rzg2-ex.dtsi
new file mode 100644
index 0000000..1bc2417
--- /dev/null
+++ b/arch/arm64/boot/dts/renesas/hihope-rzg2-ex.dtsi
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * Device Tree Source for common parts of Sub boards for HiHope RZ/G2M platform
+ * and HiHope RZ/G2N platform
+ *
+ * Copyright (C) 2019 Renesas Electronics Corp.
+ */
+
+/ {
+};
diff --git a/arch/arm64/boot/dts/renesas/r8a774a1-hihope-rzg2m-ex.dts b/arch/arm64/boot/dts/renesas/r8a774a1-hihope-rzg2m-ex.dts
index 8bee8b2..bce3676 100644
--- a/arch/arm64/boot/dts/renesas/r8a774a1-hihope-rzg2m-ex.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774a1-hihope-rzg2m-ex.dts
@@ -7,6 +7,7 @@
  */
 
 #include "r8a774a1-hihope-rzg2m.dts"
+#include "hihope-rzg2-ex.dtsi"
 
 / {
 	model = "Hoperun Technology HiHope RZ/G2M extension board \
-- 
2.7.4

